components/alert/__tests__/alert.browser-test.jsx

```diff
@@ -67,14 +67,14 @@ describe('SLDSAlert: ', function describeFunction() {
 		/* Please notice the of `function () {}` and not () => {}.
 		 * It allows access to the Mocha test context via `this`.
 		 */
-		it('calls onRequestClose handler', function() {
+		it('calls onRequestClose handler', function () {
```

Comment:
the previous formatting was better

Reply:
Yeah, this change came with the Prettier 2.0 update.